jamf_protect/manifest.json

```diff
@@ -11,7 +11,6 @@
     "description": "Endpoint security and mobile threat defense (MTD) for Mac and mobile devices.",
     "title": "Jamf Protect",
     "classifier_tags": [
-      "Supported OS::Linux",
```

Comment:
curious question, what is this repo for? I thought tags are in `integrations-internal-core`
ie https://github.com/DataDog/integrations-internal-core/blob/main/wiz/manifest.json#L14